test/expressions/generateExpressionTests.py

```diff
@@ -76,9 +76,47 @@ def parse_signature_file(sig_file):
         if not signature.endswith(")\n"):
             part_sig = signature
             continue
-        res.append(signature)
+        signature)
```

Comment:
Typo?

Reply:
Whups, yeah this is a bad find-replace my b